modules/api_loader.lua

```diff
@@ -51,8 +64,7 @@ local function LoadParamsInFunction(param, interface)
   return name, data
 end
 
-
-
+--- Load Enums values
```

Comment:
@LuxoftAKutsan,
And LDOC documentation.
Shortly: `---` is a ldoc comment; the same need to be the first line of each file to be parsed by ldoc.